utils/supabase/server.ts

```
@@ -22,4 +48,66 @@ export const createServerSupabaseClient = async () => {
       },
     },
   );
-};
+}
+
+// For middleware
+export function getSupabaseMiddleware(request: Request) {
+  let response = new Response();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'response' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)